tomviz/ImageStackModel.cxx

```diff
@@ -0,0 +1,109 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+
+#include "ImageStackModel.h"
+
+#include <QBrush>
+#include <QFileInfo>
+#include <QModelIndex>
+#include <QObject>
+#include <QString>
+
+ImageStackModel::ImageStackModel(QObject* parent,
+                                 const QList<ImageInfo>& filesInfo)
+  : QAbstractTableModel(parent), m_filesInfo(filesInfo)
+{
+}
+
+int ImageStackModel::rowCount(const QModelIndex& parent) const
```

Comment:
Unused variable causing a compiler warning, I prefer just deleting them, others like Q_UNUSED or whatever it is called.